.mergify.yml

```diff
@@ -4,23 +4,35 @@ pull_request_rules:
       - author~=dependabot\[bot\]|dependabot-preview\[bot\]
       - status-success=Travis CI - Pull Request
       - status-success=continuous-integration.azure-devops
+      - or:
```

Comment:
I don't think we need this block after all. Without it, the CI will run, the standalone will be created, mergify will "approve", then the next rule will comment, and the merge rule will not merge unless the codebuild jobs post success.

Reply:
Okay.  I think you're right.  Otherwise, there would be no approve rule for the submodules or pyinstaller.

Reply:
Fixed it.